staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -1627,13 +1620,13 @@ func TestList(t *testing.T) {
 }
 
 func TestListContinuation(t *testing.T) {
-	etcdClient := testserver.RunEtcd(t, nil)
-	codec := apitesting.TestCodec(codecs, examplev1.SchemeGroupVersion)
-	transformer := &prefixTransformer{prefix: []byte(defaultTestPrefix)}
+	ctx, store, etcdClient := testSetup(t)
+	transformer, ok := store.transformer.(*prefixTransformer)
```

Comment:
nit: I'm not going to push strongly for that, but I would personally switch to "hard conversion" here:
```
transformer := store.transformer.(*prefixTransformer)
```

It's a bad practice in production code, but here in the test where we know it should work fine it would make the test a bit more readable.

Reply:
I believe JSON output and parsing the test failures breaks down when panics occur, so I think it may be nicer to have `t.Fatal` here instead.

Reply:
Ah, nevermind - the `-json` encoder handles it correctly. I think panic due to test package timeout is what I was thinking of.